Makefile

```diff
@@ -194,6 +194,9 @@ upload-datadog-secrets: check-env-vars ## Decrypt and upload Datadog credentials
 	$(if $(wildcard ${DATADOG_PASSWORD_STORE_DIR}),,$(error Password store ${DATADOG_PASSWORD_STORE_DIR} does not exist))
 	@scripts/upload-datadog-secrets.sh
 
+upload-tracker-token: check-env-vars ## Decrypt and upload Pivotal tracker API token to S3
+	pass pivotal/tracker_token | aws s3 cp - "s3://${DEPLOY_ENV}-state/tracker_token"
+
```

Comment:
It's simple and it works as I did check. I'm not sure however, if this is the desired approach, by looking at "how are datadog secrets handled".

I'm not asking to change it, just making a note for other Reviewer, who may perhaps have an opinion on this.